packages/common/src/types/index.ts

```diff
@@ -1,8 +1,12 @@
+import { Geometry } from 'geojson';
+
 interface LogFn {
   (obj: unknown, msg?: string, ...args: unknown[]): void;
   (msg: string, ...args: unknown[]): void;
 }
 
+type BBox = [number, number, number, number];
```

Comment:
Use `BoundingBox` type from `geojson` package instead

Reply:
geojson's BBox is either `[number, number, number, number] | [number, number, number, number, number, number]`, I prefer to have more precision